src/main/java/org/folio/circulation/domain/LoanAndRelatedRecords.java

```diff
@@ -42,6 +42,10 @@ public LoanAndRelatedRecords withItem(Item newItem) {
     return withLoan(loan.withItem(newItem));
   }
 
+  public LoanAndRelatedRecords withItemLocation() {
+    return withLoan(loan.changeItemLocationAtCheckout(this.loan.getItemId()));
```

Comment:
This provides the `itemId` as the value to the parameter `itemLocationId` is that correct?